lib/oneview-sdk/resource/api300/c7000/enclosure_group.rb

```diff
@@ -26,11 +26,64 @@ def initialize(client, params = {}, api_ver = nil)
           # Default values:
           @data['type'] ||= 'EnclosureGroupV300'
           @data['stackingMode'] ||= 'Enclosure'
+          @data['enclosureCount'] ||= 1
           @data['interconnectBayMappingCount'] ||= 8
-          create_interconnect_bay_mapping unless @data['interconnectBayMappings']
```

Comment:
I guess the `@data['interconnectBayMappings']` still needs a default value like `[]`